unison-src/transcripts/patternMatchTls.md

```diff
@@ -13,13 +13,18 @@ use builtin.io2.Tls newClient send handshake terminate
 
 frank: '{IO} ()
 frank = do 
-  (Right socket) = clientSocket.impl "example.com" "443"
```

Comment:
I didn't even know we had this syntax. I'm guessing it produces an error now? What kind?

Reply:
Destructuring binds all desugar to pattern matching in the obvious way, so it'll be a pattern coverage error.